src/pages/LoanFinding/LoanFinding.vue

```diff
@@ -181,26 +193,49 @@ export default {
 			enableQFMobileVersion: false,
 			enableAlmostFundedRow: false,
 			HandOrangeIcon,
+			enableLoanRecommendations: false,
 		};
 	},
 	apollo: {
 		preFetch(config, client) {
 			return client.query({
-				query: experimentAssignmentQuery, variables: { id: THREE_LOANS_RECOMMENDED_ROW_EXP_KEY }
+				query: experimentAssignmentQuery,
+				variables: { id: THREE_LOANS_RECOMMENDED_ROW_EXP_KEY }
 			}).then(() => {
-				const recommendedLoansPromise = client.query({
-					query: flssLoansQueryExtended,
-					variables: prefetchedRecommendedLoansVariables
+				return client.query({
```

Comment:
If the pre fetches run simultaneously (see my other comment a min ago), then we can keep these queries here, but we can also use Promise.all, for example: https://github.com/kiva/ui/blob/a1f117681edcf26b6bbc5cbda05d4c79a89d1085/src/pages/Lend/LoanChannelCategoryControl.vue#L451

Reply:
done